client/components/Byline/Byline.tsx

```diff
@@ -72,6 +119,23 @@ const Byline = (props: PrivateBylineProps) => {
 		if (typeof contributors === 'string') {
 			return contributors;
 		}
+		if (bylinePrefix === BylineOption.Az || BylineOption.Za) {
+			// ReviewNote: Do we need to hande the case where a contributor is a string?
```

Comment:
Yes! We have contributors coming into this component from e.g. the `PubEdgePreview`, where we probably don't have any structured information on their name.